spyder/plugins/statusbar/plugin.py

```diff
@@ -176,3 +198,41 @@ def clock_status(self):
     def _statusbar(self):
         """Reference to main window status bar."""
         return self._main.statusBar()
+
+    def _organize_status_widgets(self):
+        """
+        Organize the status bar widgets once the application is loaded.
+        """
+        # Desired organization from

```

Comment:
```suggestion
        # Desired organization
```

I think `from` is unnecessary here.